Userland/Applications/Browser/Tab.cpp

```diff
@@ -423,20 +425,21 @@ Tab::Tab(Type type)
 
         if (g_search_engine == url_format) {
             action->set_checked(true);
+            search_engine_set = true;
         }
         action->set_status_tip(url_format);
     };
 
-    auto disable_search_engine_action = GUI::Action::create_checkable(
+    m_disable_search_engine = GUI::Action::create_checkable(
         "Disable", [this](auto&) {
             g_search_engine = {};
             auto m_config = Core::ConfigFile::get_for_app("Browser");
             m_config->write_entry("Preferences", "SearchEngine", g_search_engine);
         },
         this);
-    search_engine_menu.add_action(disable_search_engine_action);
-    m_search_engine_actions.add_action(disable_search_engine_action);
-    disable_search_engine_action->set_checked(true);
+    search_engine_menu.add_action(*m_disable_search_engine);
+    m_search_engine_actions.add_action(*m_disable_search_engine);
+    m_disable_search_engine->set_checked(true);
```

Comment:
Probably better to not drop the `_action` at the end, because it makes what the variable is meant to be clearer for readers of the code.

Reply:
Fixed